thoth/metrics_exporter/jobs.py

```diff
@@ -115,6 +115,40 @@ def count_graph_sync_job_status(job_list_items: list) -> dict:
     return graph_sync_jobs_status
 
 
+def count_job_status(job_list_items: list) -> dict:
+    """Count the number of created, active, failed, succeeded, pending Jobs."""
+    jobs_status = {}
+    job_status = ["created", "active", "failed", "succeeded", "pending", "retry", "waiting", "started"]
+
+    # Initialize
+    for job_s in job_status:
+        jobs_status[job_s] = 0
+
+    for item in job_list_items:
+        jobs_status["created"] += 1
+
+        if "succeeded" in item["status"].keys():
+            jobs_status["succeeded"] += 1
+        elif "failed" in item["status"].keys():
+            jobs_status["failed"] += 1
+        elif "active" in item["status"].keys():
+            jobs_status["active"] += 1
+        elif "pending" in item["status"].keys():
+            jobs_status["pending"] += 1
+        elif not item["status"].keys():
+            jobs_status["waiting"] += 1
+        elif "startTime" in item["status"].keys() and len(item["status"].keys()) == 1:
+            jobs_status["started"] += 1
+        else:
+            try:
+                if "BackoffLimitExceeded" in item["status"]["conditions"][0]["reason"]:
+                    jobs_status["retry"] += 1
+            except Exception as excptn:
+                _LOGGER.error("Unknown job status %r", item)
+                _LOGGER.exception(excptn)
+    return jobs_status
```

Comment:
Do we have this code somewhere? I think I saw it already :)

Reply:
A different one, because for graph-sync jobs the label is the same for several categories of graph-syncs. We can generalize the label, if we want to add more labels to be monitored in the future.

Reply:
I ll try to generalize it